srcpkgs/asahi-audio/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'asahi-audio'
+pkgname=asahi-audio
+version=2.5
+revision=1
+build_style=gnu-makefile
+depends="alsa-ucm-conf-asahi>=5 speakersafetyd>=0.1.9 linux-asahi>=6.8.9
+ pipewire>=1.0.5 wireplumber>=0.5.2 bankstown>=1.1.0 lsp-plugins-lv2>=1.2.14"
```

Comment:
are these version requirements really necessary?

Reply:
yes, see https://github.com/AsahiLinux/asahi-audio

Reply:
only lsp-plugins-lv2 should have a version requirement

Reply:
I think it doesn't hurt to be extra careful, but I don't insist, so just let me know how you want this :)

Reply:
only lv2

Reply:
Done.